.github/workflows/test-int.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test: ["AtlasProject", "AtlasDeployment", "AtlasDatabaseUser"]
+        test: ["AtlasProject","AtlasDatabaseUser", "legacy-deployment", "advanced-deployment"]
```

Comment:
```suggestion
        test: ["AtlasProject", "AtlasDatabaseUser", "AtlasLegacyDeployment", "AtlasAdvancedDeployment"]
```

Reply:
My bad. Now it makes no sense to separate tests